src/mmpars.c

```diff
@@ -5921,113 +5923,113 @@ vstring readInclude(const char *fileBuf, long fileBufOffset,
   free_vstring(fullIncludeFn);
 
   return newFileBuf;
-} /* readInclude */
-
-/* This function returns a pointer to a buffer containing the contents of an
-   input file and its 'include' calls.  'Size' returns the buffer's size.  */
-/* TODO - ability to flag error to skip raw source function */
-/* If NULL is returned, it means a serious error occurred (like missing file)
-   and reading should be aborted. */
-vstring readSourceAndIncludes(const char *inputFn /*input*/, long *size /*output*/) {
+} // readInclude
+
+// This function returns a pointer to a buffer containing the contents of an
+// input file and its 'include' calls.  'Size' returns the buffer's size.
+// TODO - ability to flag error to skip raw source function.
+// If NULL is returned, it means a serious error occurred (like missing file)
+// and reading should be aborted.
+vstring readSourceAndIncludes(const char *inputFn /* input */, long *size /* output */) {
   long i;
-/*D*//* long j; */
-/*D*//* vstring_def(s); */
+// D // long j;
+// D // vstring_def(s);
   vstring_def(fileBuf);
   vstring_def(newFileBuf);
 
   vstring_def(fullInputFn);
   flag errorFlag = 0;
 
-  /* Read starting file */
+  // Read starting file
   let(&fullInputFn, cat(g_rootDirectory, inputFn, NULL));
-  fileBuf = readFileToString(fullInputFn, 1/*verbose*/, &(*size));
+  fileBuf = readFileToString(fullInputFn, 1 /* verbose */, &(*size));
   if (fileBuf == NULL) {
     print2("?Error: file \"%s\" was not found\n", fullInputFn);
     fileBuf = "";
     *size = 0;
     errorFlag = 1;
-    /* goto RETURN_POINT; */ /* Don't go now so that g_IncludeCall[]
-         strings will be initialized.  If error, then blank fileBuf will
-         cause main while loop to break immediately after first
-         getNextInclusion() call. */
+    // goto RETURN_POINT; // Don't go now so that g_IncludeCall[]
+    // strings will be initialized.  If error, then blank fileBuf will
+    // cause main while loop to break immediately after first
+    // getNextInclusion() call.
```

Comment:
```suggestion
    // Don't go now so that g_IncludeCall[]
    // strings will be initialized.  If error, then blank fileBuf will
    // cause main while loop to break immediately after first
    // getNextInclusion() call.
    // goto RETURN_POINT; 
```